src/server/cellular.cpp

```diff
@@ -122,7 +122,11 @@ cellular_time_t::cellular_time_t(const NetworkTimeInfo &cnti)
     , ts(0)
 {
     if (cnti.isValid() and cnti.dateTime().isValid()) {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+        value = cnti.dateTime().toSecsSinceEpoch();
```

Comment:
This was introduced in 5.8 and toTime_t obsoleted at the same. As toTime_t got removed on Qt 6.0 this is perfectly fine. Take this more like a note. There's no need to start changing these version checks. 
https://doc.qt.io/qt-5/qdatetime.html#toSecsSinceEpoch